Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -115,15 +115,13 @@ class BrowserViewController: UIViewController {
     override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransitionToSize(size, withTransitionCoordinator: coordinator)
 
-        guard let displayedPopoverController = self.displayedPopoverController where displayedPopoverController.isBeingPresented() else {
```

Comment:
An easier fix might well have been just to remove the where clause from the guard statement